test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature

```diff
@@ -0,0 +1,75 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file	test.feature
+# *
+# * \test	Status Sign up
+# * \date	May 2022
+# **
+# *****************************************************************************/
+
+Feature: Status Desktop Sign Up
+
+  As a user I want to Sign-up into the Status Desktop application.
+  The following scenarios cover Sign up process.
+
+  Scenario: User signs up and signs in with password
+    Given A first time user lands on the status desktop and generates new key
+    When user inputs username tester123 and password TesTEr16843/!@00
```

Comment:
Could you parametrize them?

Reply:
Do you mean like using a scenario Outline?

Reply:
Yes